requests_oauth2/oauth2.py

```diff
@@ -23,6 +25,27 @@ def __init__(self, client_id, client_secret, site, redirect_uri, authorization_u
             self.authorization_url = authorization_url
         if token_url is not None:
             self.token_url = token_url
+        if revoke_url is not None:
+            self.revoke_url = revoke_url
+
+    def _make_request(self, url, data, headers=None):
+        """
+        Make a request to an OAuth2 endpoint
+        """
+        if headers:
+            response = requests.post(url, data=data, headers=headers)
+        else:
+            response = requests.post(url, data=data)
+
+        if isinstance(response.content, basestring):
```

Comment:
Since we're targeting Python 3 support now as well, referencing `basestring` won't work.